### PR TITLE
Flashbangs no longer work on the dead

### LINKED ISSUE #14569: Flashbang still blinds and makes your ear ring if you are dead but inside your body
Someone throws a flashbang
You are dead but inside your body
You still get fully affected by the flashbang

### PR DESCRIPTION
**Pull request by silicons**
Fixes #14569

### CODE REVIEW

File: code/game/objects/items/weapons/grenades/flashbang.dm [hunk 1]
```diff
@@ -19,6 +19,8 @@
 	qdel(src)
 
 /obj/item/weapon/grenade/flashbang/proc/bang(turf/T , mob/living/M)
+	if(M.stat)	//They're unconscious!
```
Comment (AnturK) [inline code/game/objects/items/weapons/grenades/flashbang.dm]: stat is not boolean use proper defines here.

**Comment by coiax** [comment]
I'm pretty sure even unconscious people will get blinded and deafened. Closing your eyes is no defence.

**Comment by kevinz000** [comment]
ok

**Comment by ShadowDeath6** [comment]
Nice comment very informational :+1:

**Pull request merged**